### DISCUSSION TITLE
Visual Studio 2022 ARM Support

### DISCUSSION THREAD

**Discussion by OnlyBMan** [community/community#39628] [Copilot Conversations] [answered]
Hello! Since Microsoft has recently released the ARM version of Visual Studio 2022, I was wondering if GitHub would release a ARM64 extension for Copilot. Thank you!

**Comment by Akash1134** [comment]
Hi @OnlyBMan , Thanks for posting on GitHub Communities. I have shared your feedback with the concerned team. I'll keep you informed once I get an update on it.

**Comment by LukeStephen** [comment]
Any update on this question? Any alpha bits I can help test?

**Reply by ayim** [reply]
Released yesterday - version 1.116.0.0 will install on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.

**Reply by ayim** [reply]
@LukeStephen ^

**Comment by samalone** [comment]
I want to request this as well. While my desktop workstation is Intel, my laptop is ARM (MacBook Pro). VS 2022 runs great on it, but I was disappointed that I can only run Copilot on my desktop.

**Reply by ayim** [reply]
Request granted... version 1.116.0.0 will install on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by ayim** [reply]
@samalone ^

**Comment by Yevhen-Myroshnychenko** [comment]
I wonted to say that it would be nice, if you will add support for `Visual Studio 2022 for Mac M1`, and also if you will add support for  `Visual Studio 2022 for Windows 11, ARM`. Thank you!

**Reply by ayim** [reply]
Yesterday's release of the code completions extension (version 1.116.0.0) will work on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Comment by malaksedarous** [comment]
Is there any update on this? I can't be the only one in the developer community using ARM Surfaces that want to try Copilot X

**Reply by ayim** [reply]
You and others using ARM Surfaces will be able to use yesterday's release (version 1.116.0.0) on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Comment by ronmichael** [comment]
I'll chime in and support @malaksedarous's comments. I'm paying for Copilot but haven't been able to use it for a month or so since I switched to VS ARM. *And from what I've seen updating other extensions to support ARM, it may not be that much effort.

**Reply by maxwell-fraser** [reply]
I am in the same boat as you.

**Reply by ayim** [reply]
All aboard - yesterday's release (version 1.116.0.0) will work on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by ayim** [reply]
@ronmichael @maxwell-fraser ^

**Comment by jedibatman** [comment]
This is, quite frankly, pretty ridiculous that it is taking soo long to support arm for a _**paid service**_. It can't possibly be so tied to the ~~extension~~ os architecture, because I can now see a _preview_ version for a visual studio extension that _**does support**_ arm (https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat).

GitHub / Microsoft should have no business rolling out preview arm extensions when they don't even support it for the one _paid for_ service.

smh

**Reply by ayim** [reply]
Rolling out ... yesterday's release of the code completions extension (version 1.116.0.0) will work on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by ayim** [reply]
@jedibatman ^

**Comment by maxwell-fraser** [comment]
Yep, the repview version is there but that does not work for me, I get an
error basically telling me I need to sign up for the GitHub Copilot chat
Private Preview which I believe is the next iteration they started
promoting.

**Comment by softgandhi** [comment]
https://developercommunity.visualstudio.com/t/Can-we-have-Github-Copilot-support-for-V/10325883?page=21&pageSize=15&sort=active&topics=Azure

Request to upvote this feature so that Microsoft can hear the user voice loudly.

**Reply by ayim** [reply, marked as answer]
Shipped. Yesterday's release (version 1.116.0.0) will work on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by ayim** [reply]
@softgandhi ^

**Comment by Emilprivate** [comment]
Would also love to see ARM support

**Reply by ayim** [reply]
Good morning - yesterday's release (version 1.116.0.0) will work on ARM.
If you encounter any experiences where we can improve stability and experience on ARM, please let us know.
https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by ayim** [reply]
@Emilprivate ^
